src/modules/profile/graphql/fragments/creatorNotificationsSettingsFragment.ts

```diff
@@ -0,0 +1,23 @@
+import { gql } from '@apollo/client'
+
+export const FRAGMENT_CREATOR_NOTIFICATIONS_SETTINGS = gql`
+  fragment CreatorNotificationsSettings on CreatorNotificationSettings {
+    userId
+    project {
+      id
+      title
```

Comment:
create a new fragment for this project, it'll be easy to type the project you end up using later for creator notification settings.

Reply:
I have change the implementation of CreatorNotifications, let me know if you think it's still worth it. The project it self is not very relevant